txprocessor/zkinputsgen_test.go

```diff
@@ -904,7 +904,7 @@ func TestZKInputs6(t *testing.T) {
8415507397039526176679688506579114602308891419321366426899227563735979799040","6297316909055097129174836401224799012703267232549005289288013654975501958320","0","0","0","0","0","0","0","0","0","0","0","0","0"],["10654609496535964558909566290432480298919624370744901502569342519550073424476","11872834193405429155785221606635733041877785946183895548055231228630415671526","13338303469345047684537862475562058883161632564586987947839572622273990170507","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"]],"siblings3":[["10481589107625349543356999071414880624641854424821273757413664142067964978076","11872834193405429155785221606635733041877785946183895548055231228630415671526","752163647728876524378487984296494416527655885600573772722793563230109953871","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["10481589107625349543356999071414880624641854424821273757413664142067964978076","9511566748286243410718405432244363254186644304849399669411385661138784704473","18183543918182770941889882117666634479723575558022332914228723866339567730166","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"],["0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0"]],"sign1":["0","0","0","0","0","0","0","0","0","0","0"],"sign2":["0","0","0","0","0","0","0","0","0","0","0"],"sign3":["0","0","0","0"],"toBjjAy":["0","0","0","0","0","0","0","0","0","0","0"],"toEthAddr":["0","0","0","0","0","0","0","0","0","0","0"],"toIdx":["259","256","257","1","0","0","0","0","0","0","0"],"tokenID1":["0","0","1","0","0","0","0","0","0","0","0"],"tokenID2":["0","0","1","0","0","0","0","0","0","0","0"],"tokenID3":["1","0","0","0"],"txCompressedData":["13269348750238205523937671488142474492700876111663723512368426903055","13269348750238205523937671488142474492463191624120932751387608737295","13269348750238205523937693788887673023165561322353469174392705246735","13269348750238301304908975606196121868949207077307500584429224191503","3322668559","3322668559","3322668559","3322668559","3322668559","3322668559","3322668559"],"txCompressedDataV2":["51833393555617990327881529250556548829772431193964427068844278016","51833393555617990327881529250556544868364305480746702966716629256","51833393555617990327881616362842480590019077817863366651552989446","51833393555618364472300685961703608973089606562344838443583930624","0","0","0","0","0","0","0"]}` //nolint:lll
```

Comment:
@arnaucube set these tests vectors originally.  I know that he verified their validity manually with snarkjs.  I don't know if you made this verification, or if @arnaucube has checked this already; I want to hear the opinion of @arnaucube before approving this change.

Reply:
Yes, yesterday I checked locally this new two values of the test vectors with the circom circuits and it worked fine. I commented it on yesterday's daily meeting, and we set a workshop next Monday to make the knowledge transfer about how to test the Go generated ZKInputs with the Circom Hermez circuits, for future updates in the hermez-node that affects this part.